### ISSUE TITLE
Network Health 2.0 Binding: define ping retries

### ISSUE THREAD

**Issue by gersilex** [openhab/openhab-addons#100]
For checking the Network Health of low-power wifi devices like smartphones it would be nice to be able to **specify a ping retry count**. Therefore it should not only ping the device once, but retry the ping _n times_ after timing out the previous ping.

This is the reason:
- Android smartphones sometimes stop network communications for a couple of seconds. This means it will not receive the ping at all and causes it to definitely time out. This happens most on timeouts < 60 seconds but cannot be fixed by setting the timeout higher.
- Apple devices completely turn off their wifi interface while the screen is off and the charger is not connected. All polling or TCP Listeners are using the mobile network. 

Flow:

```
Variables: <IP> [<tries=1>] [<timeout=5000>] [<port=0>]
- Ping <IP> with <timeout> with either ICMP (if port=0) or TCP (with port > 0)

On failure:
- increment tries counter and if counter < <tries>, jump up to retry
else
- send command OFF

On success:
- send command ON
- break
```

Should be pretty clear and if the library supports it, we don't even have to implement it and only implement the handling of the new variables.

Unfortunately I'm not experienced with Java. Great work so far!

**Comment by kaikreuzer** [comment]
@TecSet, is this something you would want to enhance the Network Health binding with?

**Comment by TecSet** [comment]
Definitely, it sounds interesting. I'm going to take a look at it.

**Comment by kaikreuzer** [comment]
Implemented with #148
